binderhub/builder.py

```diff
@@ -137,21 +137,33 @@ def escape(s):
         })
 
     async def get(self, provider_prefix, spec):
-        """Get a built image for a given GitHub user, repo, and ref."""
-        # We gonna send out event streams!
+        """Get a built image for a given spec (user, repo, and ref).
```

Comment:
This method actually launches a given repo of a given provider, building it if needed

The 'get' is tornado convention since it's a GET request.